Waffle March 2

Thinking-waffle opened the pull request.

Thinking-waffle committed:
improved RAJ - Maharajahs characters

Thinking-waffle committed:
Updated gain_random_agency_upgrade

Thinking-waffle committed:
Update netherlands_MtG.txt

Thinking-waffle committed:
Update netherlands.txt

Thinking-waffle committed:
following paradox Dutch changes a year later...

Thinking-waffle committed:
Update canada.txt

katsuragiyy committed:
Korean shenanigans

Thinking-waffle committed:
Superior firepower army spirit reduce cost of rocket arty too

Thinking-waffle committed:
fixed mistakes conflicting with Eastern front plane pack units_planes.asset

marijn2112 commented on common/military_industrial_organization/organizations/HOL_organization.txt:
Should adjust the focuses too

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-march-2

marijn211 committed:
Fix Bosnia releasing in Yalta

Thinking-waffle committed:
Update r56_units_tanks.asset

Thinking-waffle committed:
Update 00_tank_icons.txt

marijn211 committed:
Fix icon and hard to explain bugfix

Thinking-waffle committed:
Fix r56 india fascist civil war part 1

Thinking-waffle committed:
fixing r56_india fascist civil war part 2

- should take into account agrarian reform
-added starting army air and naval XP and command power


TO DO : check the RAJ_create_generals = yes effect
fix the namelist

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Fix German aces of the focus tree

should be adjusted for historicity's sake

Thinking-waffle committed:
Update 00_aces.txt

Thinking-waffle committed:
Update romania.txt

Thinking-waffle committed:
Update romania.txt

Thinking-waffle committed:
Update poland.txt

Thinking-waffle committed:
Frogmen adjustment canada.txt

Thinking-waffle committed:
Update france.txt

Thinking-waffle committed:
Update equipment_l_english.yml

Thinking-waffle committed:
Add files via upload

Thinking-waffle committed:
No anti communist intervention for communist USA 10_toa_on_actions.txt

Thinking-waffle committed:
added an early game ace for Japan

Thinking-waffle committed:
Siamese "aces" (pilots with one kill).

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Polish ace names

Thinking-waffle committed:
Japanese ace names

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Update SOV - Soviet union.txt

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Update aces.gfx

Thinking-waffle committed:
improving bba japanese icon choice, to be continued

Thinking-waffle committed:
Update 00_plane_icons.txt

Thinking-waffle committed:
ending adjustment of Japanese small plane 2d icons

Thinking-waffle committed:
I lied about it being the end...

Thinking-waffle committed:
Update JAP - Japan.txt

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Update GER - Germany.txt

Thinking-waffle committed:
Update GER - Germany.txt

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Update SOV - Soviet union.txt

Thinking-waffle committed:
Update 00_names.txt

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-march-2

marijn211 reviewed (changes requested):
A bunch of icon errors, just a selection:

```
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with NZL medium_tank_aa_chassis_0
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with NZL medium_tank_aa_chassis_1
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_C", associated with NZL medium_tank_aa_chassis_2
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_A", associated with NZL medium_tank_aa_chassis_3
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with AST medium_tank_aa_chassis_0
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with AST medium_tank_aa_chassis_1
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_C", associated with AST medium_tank_aa_chassis_2
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_A", associated with AST medium_tank_aa_chassis_3
[12:44:23][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_heavy_fighter_equipment_2_entity", associated with YUG medium_plane_fighter_airframe_2
[12:44:23][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_heavy_fighter_equipment_2_entity", associated with YUG medium_plane_fighter_airframe_3
[12:44:23][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_heavy_fighter_equipment_2_entity", associated with YUG medium_plane_fighter_airframe_1
[12:44:23][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_fighter_equipment_1_entity", associated with YUG small_plane_airframe_2
[12:44:23][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_fighter_equipment_3_entity", associated with YUG small_plane_airframe_2
[12:44:23][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_fighter_equipment_1_entity", associated with YUG small_plane_airframe_3
[12:44:23][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_fighter_equipment_3_entity", associated with YUG small_plane_airframe_3
[12:44:23][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_fighter_equipment_3_entity", associated with YUG small_plane_airframe_1
[12:44:23][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_fighter_equipment_1_entity", associated with YUG small_plane_airframe_1

[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with CAN medium_tank_aa_chassis_0
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with CAN medium_tank_aa_chassis_1
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_C", associated with CAN medium_tank_aa_chassis_2
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_A", associated with CAN medium_tank_aa_chassis_3

[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with SAF medium_tank_aa_chassis_0
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with SAF medium_tank_aa_chassis_1
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_C", associated with SAF medium_tank_aa_chassis_2
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_A", associated with SAF medium_tank_aa_chassis_3
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with MAL medium_tank_aa_chassis_0
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_B", associated with MAL medium_tank_aa_chassis_1
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_C", associated with MAL medium_tank_aa_chassis_2
[12:44:23][no_game_date][equipment_graphic_database.cpp:36]: GFX referenced in equipment graphic database does not exist: "GFX_ENG_medium_AA_A", associated with MAL medium_tank_aa_chassis_3
```

Thinking-waffle committed:
Update 00_plane_icons.txt

Thinking-waffle committed:
Update 01_bba_plane_icons.txt

Thinking-waffle committed:
Update tank_profiles.gfx

Thinking-waffle committed:
Update 00_tank_icons.txt

Thinking-waffle committed:
Update HOL - Holland.txt

Thinking-waffle committed:
Update HOL.txt

Thinking-waffle committed:
Update HOL.txt

Thinking-waffle committed:
Update HOL.txt

Thinking-waffle committed:
added characters LIB.txt need to update the history file

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB - Liberia.txt

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-march-2

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB - Liberia.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update 00_traits.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB - Liberia.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Update LIB - Liberia.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
Added Pola VP - 852-Istria.txt

Thinking-waffle committed:
Update r56_austria.txt

Thinking-waffle committed:
Istrian VPs (Pola and Fiume)

Thinking-waffle committed:
Update 852-Istria.txt

Thinking-waffle committed:
fixed wrong event when trying to join the axis for scotland.txt

Thinking-waffle committed:
Update LIB.txt

Thinking-waffle committed:
upgrade anarchist spying potential

Thinking-waffle committed:
Update argentina.txt

Thinking-waffle committed:
should avoid the AI needlessly reducing mobilization stance

Thinking-waffle committed:
Experimental AI guidelines for r56i_laws_gender.txt

Telling the AI to use gender laws

Thinking-waffle committed:
experimental AI guidelines for r56i_laws_leadership.txt

Thinking-waffle committed:
Ai guidlines for r56i_laws_social.txt

Thinking-waffle committed:
Update r56i_laws_gender.txt

Thinking-waffle committed:
fix r56i_laws_leadership.txt

Thinking-waffle committed:
Update land_doctrine.txt

Thinking-waffle committed:
ai guidelines for _economic.txt

Thinking-waffle committed:
fix typo LIB.txt

Thinking-waffle committed:
Update r56_iceland.txt

Thinking-waffle committed:
Update ICE_l_english.yml

Thinking-waffle committed:
Update iceland.txt

Thinking-waffle committed:
Update r56_iceland.txt

Thinking-waffle committed:
Bananas for Iceland.

Thinking-waffle committed:
Update iceland.txt

Thinking-waffle committed:
Update r56_iceland.txt

Thinking-waffle committed:
Update GER decision forming the grand duchy of Scotland

Thinking-waffle committed:
added SS decision file to fix Belgium state error

Thinking-waffle committed:
Update SS.txt

Thinking-waffle committed:
Adding the option for Karl Pius to LAR_Spain.txt but is it enough?

Thinking-waffle committed:
mostly reverted the change LAR_Spain.txt

Thinking-waffle committed:
attempt to fix crash in civil war in  event Austria.104

I hope PREV works here, otherwise rewrite the scope, put it out of start_civil_war as I think that that's what caused the crash

Thinking-waffle committed:
attempt to fix option A in austria.104

Thinking-waffle committed:
add dam for event about building a dam. LAT_events.txt

Thinking-waffle committed:
Update LAT_events.txt

Thinking-waffle committed:
Update LAT_events.txt

TheBrit7 committed:
Liechtenstein portraits

Thinking-waffle committed:
removed generic MIO for PRU

Thinking-waffle committed:
fix wrong trait for artillery MIO PRU_organization.txt

Thinking-waffle committed:
Carlos I of Peru doesn't forget to hate communism peru.txt

Thinking-waffle committed:
Update r56_austria.txt

Thinking-waffle committed:
Sent PRU navy chief with high command traits to high command

Thinking-waffle committed:
Karl Pius character SPR.txt, impacts Peru

Thinking-waffle committed:
Removed karl pius from the history/countries PRU - Peru.txt

Thinking-waffle committed:
added Karl Pius + helicopters SPR - Spain.txt

Thinking-waffle committed:
Should fix Karl Pius in peru.txt

Thinking-waffle committed:
Update LAR_Spain.txt

Thinking-waffle committed:
Update LAR_NewsEvents.txt add karl pius flag

Thinking-waffle committed:
Update peru.txt

Thinking-waffle committed:
update karl pius and Vasyl Vyshyvanyi WW_Hungary.txt

Thinking-waffle committed:
Update hungary_wuw.txt

Thinking-waffle committed:
Update army_spirits.txt

Thinking-waffle committed:
Update Ukraine.txt

Thinking-waffle committed:
Update Ukraine.txt

Thinking-waffle committed:
Update hungary_wuw.txt

Thinking-waffle committed:
Update peru.txt

Thinking-waffle committed:
Update peru.txt

Thinking-waffle committed:
Update hungary_wuw.txt

Thinking-waffle committed:
Update HUN.txt

Thinking-waffle committed:
better focus icon picks spain.txt

Thinking-waffle committed:
Update spain.txt

Thinking-waffle committed:
more icon, oh I added a Carlist foucs in a previous commit spain.txt

Thinking-waffle committed:
Update spain.txt

Thinking-waffle committed:
Update PAP - Papal States.txt

Thinking-waffle committed:
Update spain.txt

Thinking-waffle committed:
Update spain.txt

Thinking-waffle committed:
Update spain.txt

TheBrit7 committed:
VEN

Thinking-waffle committed:
Update ship_hull_submarine.txt

marijn211 reviewed (changes requested):
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_modern_armor_entity", associated with SOV modern_tank_chassis
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_3_entity", associated with SOV small_plane_cas_airframe_1
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_2_entity", associated with SOV small_plane_cas_airframe_1
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_1_entity", associated with SOV small_plane_cas_airframe_1
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_3_entity", associated with SOV small_plane_cas_airframe_2
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_1_entity", associated with SOV small_plane_cas_airframe_2
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_2_entity", associated with SOV small_plane_cas_airframe_2
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_1_entity", associated with SOV small_plane_cas_airframe_3
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_2_entity", associated with SOV small_plane_cas_airframe_3
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_3_entity", associated with SOV small_plane_cas_airframe_3
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "GFX_SOV_basic_heavy_tank_medium", associated with SOV heavy_tank_chassis_1
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_fighter_equipment_2_entity", associated with SOV small_plane_airframe_2
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "GFX_SOV_basic_heavy_tank_medium", associated with SOV heavy_tank_chassis_0
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_fighter_equipment_2_entity", associated with SOV small_plane_airframe_3
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_fighter_equipment_2_entity", associated with SOV small_plane_airframe_1
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_2_entity", associated with SOV cv_small_plane_naval_bomber_airframe
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_3_entity", associated with SOV cv_small_plane_naval_bomber_airframe
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_1_entity", associated with SOV cv_small_plane_naval_bomber_airframe
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_super_heavy_armor_entity", associated with SOV super_heavy_tank_chassis
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_nav_bomber_equipment_1_entity", associated with SOV small_plane_naval_bomber_airframe
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_2_entity", associated with SOV small_plane_naval_bomber_airframe
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_3_entity", associated with SOV small_plane_naval_bomber_airframe
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_1_entity", associated with SOV small_plane_naval_bomber_airframe
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_3_entity", associated with SOV cv_small_plane_cas_airframe_1
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_2_entity", associated with SOV cv_small_plane_cas_airframe_1
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_1_entity", associated with SOV cv_small_plane_cas_airframe_1
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_3_entity", associated with SOV cv_small_plane_cas_airframe_2
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_1_entity", associated with SOV cv_small_plane_cas_airframe_2
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_2_entity", associated with SOV cv_small_plane_cas_airframe_2
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_1_entity", associated with SOV cv_small_plane_cas_airframe_3
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_2_entity", associated with SOV cv_small_plane_cas_airframe_3
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "SOV_CAS_equipment_3_entity", associated with SOV cv_small_plane_cas_airframe_3

[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "HUN_heavy_fighter_equipment_3_entity", associated with HUN medium_plane_fighter_airframe_1
[21:45:55][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "HUN_heavy_fighter_equipment_2_entity", associated with HUN medium_plane_fighter_airframe_1

[21:45:56][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_heavy_fighter_equipment_2_entity", associated with YUG medium_plane_fighter_airframe_2
[21:45:56][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_heavy_fighter_equipment_2_entity", associated with YUG medium_plane_fighter_airframe_3
[21:45:56][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "YUG_heavy_fighter_equipment_2_entity", associated with YUG medium_plane_fighter_airframe_1

marijn211 committed:
Fix the only non-entity error

Thinking-waffle committed:
Update plane - MIO GER - Germany.txt

Thinking-waffle committed:
correcting misreading of things from 1939 GER - Germany.txt

Thinking-waffle committed:
Update GER_1936_air_bba.txt

Thinking-waffle committed:
Add files via upload

Thinking-waffle committed:
Update military_police.txt

TheBrit7 committed:
Super secret and funny

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-march-2

marijn211 committed:
Revert "Update military_police.txt"

"penal_infantry" isn't a category

marijn211 committed:
Fix some easy to resolve errors

marijn211 committed:
Remove duplicate fix

marijn211 committed:
Fix some atrocious indentation on Peru

marijn211 committed:
Revert partial fix

marijn211 committed:
Replace banana date check with tech check
